tests/e2e/accessCustodialUser.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable linebreak-style */
+require('../../utilities/seleniumKeepAlive');
+
+module.exports = {
+  '@tags': ['qae222', 'clinician', 'parallel'],
+  'Clinician User Logs in with Existing Credentials'(browser) {
```

Comment:
Later I realised, that this is not test, but test-step, right? Ok, but I still think that each step should have assertion. Example: If login doesn't work, next test-step will fail. Because `clinicWorkspace.waitForElementVisible('@title', browser.globals.elementTimeout);` fails due it is not on correct page. So you will debug `Clinic workspace selection`, but problem will be on on `Clinician User Logs in with Existing Credentials`. 

WDYT?

Reply:
addressed in 9619963dab36e9f34fd9c99c5e7c147b76e4dc39